src/pudl_archiver/frictionless.py

```diff
@@ -163,3 +175,32 @@ def new_datapackage(
             description=data_source.description,
             version=version,
         )
+
+    @classmethod
+    def mecs(cls, resources: Iterable[Resource], version: str | None):
+        """Hack method to create a Datapackage for EIA MECS data not in PUDL metadata."""
+        return DataPackage(
+            name="MECS",
+            title="EIA MECS data",
+            sources=[
+                {
+                    "title": "EIA MECS data",
+                    "path": "https://www.eia.gov/consumption/manufacturing/data/2018/",
+                }
+            ],
+            licenses=[
+                License(
+                    **{
+                        "name": "CC-BY-4.0",
+                        "title": "Creative Commons Attribution 4.0",
+                        "path": "https://creativecommons.org/licenses/by/4.0",
+                    }
+                )
+            ],
+            resources=sorted(resources, key=lambda x: x.name),  # Sort by filename
+            contributors=[CONTRIBUTORS["catalyst-cooperative"]],
+            created=str(datetime.utcnow()),
```

Comment:
```suggestion
            created=str(datetime.datetime.now(tz=datetime.UTC).isoformat()),
```
Should mirror what we do above, this currently returns an error for me when re-running the MECS archiver with an fsspec backend.